### DISCUSSION TITLE
Switch colours for dark theme?

### DISCUSSION THREAD

**Discussion by 8Gaston8** [simibac/ConfettiSwiftUI#34] [General]
Hello, I’m struggling to make the confetti colours switch with light/dark modes. I can make it switch but it requires quitting the app and relaunching. Any tips on how to do this? For now the confetti colour was a variable which I made dependent on the light or dark mode but it’s only half working as I described. 

Thanks!

**Comment by simibac** [comment]
Have you tried setting the desired colors based on the colorScheme environment key: https://www.hackingwithswift.com/quick-start/swiftui/how-to-detect-dark-mode

**Reply by 8Gaston8** [reply]
Yes, this is the way I do it but it doesn't work.
